arm-network/2017-06-01/swagger/serviceCommunity.json

```diff
@@ -88,11 +88,11 @@
           },
           "description": "The prefixes that the bgp community contains."
         },
-		"isAuthorizedToUse": {
+        "isAuthorizedToUse": {
```

Comment:
@akshaysngupta If the spec is correct, you may need to update the example, if the example is correct, then it's mismatching the spec.